tutorials/io/70_reading_eyetracking_data.py

```diff
@@ -0,0 +1,163 @@
+# -*- coding: utf-8 -*-
+r"""
+.. _tut-importing-eyetracking-data:
+
+=======================================
+Importing Data from Eyetracking devices
+=======================================
+
+Eyetracking devices record a persons point of gaze, usually in relation to a
+screen. Typically, gaze position (also referred to as eye or pupil position)
+and pupil size are recorded as separate channels. This section describes how to
+read data from supported eyetracking manufacturers.
+
+MNE-Python provides functions for reading eyetracking data. When possible,
+MNE-Python will internally convert and store eyetracking data according to an
+SI unit (for example radians for position data, and meters for pupil size).
+
+.. seealso:: Some MNE functions may not be available to eyetracking and other
+             physiological data, because MNE does not consider them to be data
+             channels. See the :doc:`glossary </glossary>` for more
+             information.
+
+.. _import-eyelink_asc:
+
```

Comment:
we could add that there is the alternative to load in data some other way and then use `mne.preprocessing.eyetracking.set_channel_types_eyetrack`.

this could just be a note for now.. e.g. 
```
.. note:: if you have eye tracking data in a format that MNE does not support, you can try reading it using other tools and create an MNE object from a numpy array. then you can use :func:`mne.preprocessing.eyetracking.set_channel_types_eyetrack` to assign the correct channel types
```

Reply:
see [9157d98](https://github.com/mne-tools/mne-python/pull/11152/commits/9157d98b59c9ee403ed6995c05e42a3fa55ab773)